pkg/qliksense/fetch.go

```diff
@@ -29,7 +31,7 @@ func fetchAndUpdateCR(qConfig *qapi.QliksenseConfig, version string) error {
 
 	if repo, err := kapis_git.CloneRepository(destDir, QLIK_GIT_REPO, nil); err != nil {
 		return err
-	} else if err = kapis_git.Checkout(repo, version, version, nil); err != nil {
+	} else if err = kapis_git.Checkout(repo, version, fmt.Sprintf("%v-%v", version, uuid.New().String()), nil); err != nil {
```

Comment:
can we have word `by-operator` in the name, only uuid may confused with commit hash. ???

Reply:
@ffoysal: Like that: `fmt.Sprintf("%v-by-operator-%v", version, uuid.New().String())` ?

Reply:
@ffoysal: done